auth0/src/main/java/com/auth0/android/request/internal/RequestFactory.java

```diff
@@ -43,6 +43,8 @@
 
 public class RequestFactory {
 
+    public static final String DEFAULT_LOCALE_IF_MISSING = "en_US";
```

Comment:
Is this really the best place for this?

Reply:
@cocojoe Yes. It's the only class that uses it, and we have locale tests both for this class and for the class that uses it (api client). This constant is only needed because of the API client tests. I could remove those locale tests and just keep these ones, but it's nice to test the final outcome as well (i.e. request to /userinfo has locale in header). Must be public because the location in the package doesn't provide visibility otherwise.